packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -125,23 +155,29 @@ const Autocomplete = <
   HasMultipleChoices extends boolean | undefined,
   IsCustomValueAllowed extends boolean | undefined
 >({
+  errorMessage,
   hasMultipleChoices,
+  id: idOverride,
   isCustomValueAllowed,
   isDisabled,
   isLoading,
   isOptional = false,
   isReadOnly,
   hint,
   label,
+  name: nameOverride,
+  onBlur,
   onChange,
   onInputChange,
+  onFocus,
   options,
   value,
   testId,
 }: AutocompleteProps<OptionType, HasMultipleChoices, IsCustomValueAllowed>) => {
   const renderInput = useCallback(
     ({ InputLabelProps, InputProps, ...params }) => (
       <Field
+        errorMessage={errorMessage}
         fieldType="single"
         hasVisibleLabel
         id={InputLabelProps.htmlFor}
```

Comment:
for some reason if I change this to `idOverride` I get an error: `Form elements must have labels`

Reply:
That's fine. I think the label inside the `Autocomplete` is the one that creates the `id`. `InputLabelProps.htmlFor` so that label. So the `id` should go on `Autocomplete` only. It will propagate from the 2 other render functions.